manifests/charts/gateway/templates/role.yaml

```diff
@@ -0,0 +1,23 @@
+{{/*Set up roles for Istio Gateway. Note required for gateway-api*/}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "gateway.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
+rules:
```

Comment:
Do we still need this ? Can we add an option to not create this role / binding ?

Reply:
We need it if they are using Istio Gateways in another namespace only. For k8s gateway, or even Istio gateway if they are all in istio-system, its not required.

Reply:
So maybe we can remove. or at least option